include/hal/library/responder/csrlib.h

```diff
@@ -48,6 +48,48 @@ extern bool libspdm_gen_csr(uint32_t base_hash_algo, uint32_t base_asym_algo, bo
                             uint8_t *opaque_data, uint16_t opaque_data_length,
                             size_t *csr_len, uint8_t *csr_pointer,
                             bool is_device_cert_model);
+
+/**
+ * Gen CSR, which is uesd for SPDM 1.3
+ *
+ * @param[in]      base_hash_algo        Indicates the signing algorithm.
+ * @param[in]      base_asym_algo        Indicates the hash algorithm.
```

Comment:
` * @param[in]      base_hash_algo        Indicates the signing algorithm.`
` * @param[in]      base_asym_algo        Indicates the hash algorithm.`

the above two comments were mismatched, should be as following:
` * @param[in]      base_hash_algo        Indicates the hash algorithm.`
` * @param[in]      base_asym_algo        Indicates the signing algorithm.`

Reply:
Thanks. I have changed the code.